catalog/validator/catalog-validator-metacardduplication/src/main/java/org/codice/ddf/validator/metacard/duplication/DuplicationValidator.java

```diff
@@ -255,31 +332,16 @@ private SourceResponse query(Set<Attribute> attributes) {
     return response;
   }
 
-  private ValidationViolation createViolation(
-      final Set<String> attributes, Set<String> duplicates, ValidationViolation.Severity severity) {
-
-    return new ValidationViolationImpl(
-        attributes,
-        String.format(
-            "Duplicate data found in catalog: {%s}, based on attributes: {%s}.",
-            collectionToString(duplicates), collectionToString(attributes)),
-        severity);
-  }
-
   private String collectionToString(final Collection collection) {
 
     return (String)
         collection.stream().map(Object::toString).sorted().collect(Collectors.joining(", "));
   }
 
-  private Optional<MetacardValidationReport> getReport(final Set<ValidationViolation> violations) {
-    if (CollectionUtils.isNotEmpty(violations)) {
-      final MetacardValidationReportImpl report = new MetacardValidationReportImpl();
-      violations.forEach(report::addMetacardViolation);
-      return Optional.of(report);
-    }
-
-    return Optional.empty();
+  @Override
+  public DeleteRequest process(DeleteRequest input) {
```

Comment:
✏️ I feel like it's easier to read if we keep the `process` methods together

Reply:
I followed the MetacardValidityMarkerPlugin class in structure and they also had the delete process method separate to help focus on what is happening with the other process methods instead. But I can change if it makes sense to others to have it closer.

Reply:
It's just how I like it, you can leave it how it is if you prefer it 😄